### PR TITLE
Removedmp tpyaml

### PR DESCRIPTION
**Pull request by bhanuchanderpathui**
Removedmp tp yaml file

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -8,7 +8,7 @@ plugins {
 
 
 group = 'com.techsophy.tsf'
-version = '1.2.0'
+version = '1.2.1'
```
Comment (VaibhavTechsophy) [inline build.gradle:11]: no need to change the version

**Review by VaibhavTechsophy — CHANGES_REQUESTED** [review]
sorry why this file is removed?

File: build.gradle (hunk 1 above)
Comment (bhanuchanderpathui) [reply build.gradle:11]: Okay vaibhav, will revert it back to old version.

Comment (bhanuchanderpathui) [reply build.gradle:11]: @VaibhavTechsophy , as per the Description of the ticket i have removed file.

**Pull request merged**